Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1968,6 +1970,8 @@ private enum ExceptionMode {
                             attached.Set();
                             breakpointHit.Set();
                         }
+
+                        continueEvent.WaitOne(1000);

```

Comment:
You only wait a second here and then continue.  Would be better to Assert and fail the test if the command didn't run in the amount of time expected